### PR TITLE
bindata removed

### PR DESCRIPTION
**Pull request by gayathrimenath-ibm**
bindata changed to embed
Makefile changes

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -19,24 +19,11 @@ check: | verify test-unit
...
-# $2 - input dirs
-# $3 - prefix
-# $4 - pkg
-# $5 - output
-$(call add-bindata,generated,./assets/...,assets,generated,pkg/generated/bindata.go)
+.PHONY: build-image
+build-image: all
+	docker build -t $(IMAGE_REGISTRY)/origin-ibm-vpc-block-csi-driver-operator -f Dockerfile .
+	docker push $(IMAGE_REGISTRY)/origin-ibm-vpc-block-csi-driver-operator:latest
+
```
Comment (jsafrane) [inline Makefile]: Why do you remove `call build-image`? We use it to build images in our build system that does not have docker installed.

File: pkg/operator/starter.go [hunk 2]
```diff
@@ -78,15 +78,15 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		configInformers,
 	).WithCSIDriverControllerService(
 		"IBMBlockDriverControllerServiceController",
-		generated.MustAsset,
+		assets.MustAsset,
```
Comment (jsafrane) [inline pkg/operator/starter.go]: If you bump library-go, `assets.Asset` should be enough.

File: Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.15.12 as builder
+FROM golang:1.16.3 as builder
```
Comment (jsafrane) [inline Dockerfile:1]: On line 11: do not call `go mod vendor` during build. That gives you a different `vendor/` than in this repo and can lead to surprises and frustration why a wrong lib is used when `vendor` has the right one.

Comment (jsafrane) [reply Dockerfile:1]: On line 12 below: can you just call `make`? There should be only one canonical way how to build the binary.

**Review by akgunjal — APPROVED** [review]
LGTM

**Pull request merged**